### ISSUE TITLE
[Bug] cannot send '<' character to a text widget

### ISSUE THREAD

**Issue by cyberic99** [jean-emmanuel/open-stage-control#562]
When I try to send ``a<b`` to a text widget, my text is truncated as ``a``

I can still send a ``<`` using ``&gt;``

Is it expected?

**Comment by jean-emmanuel** [comment]
Nope, it happens because the value is not sanitized properly before being displayed, I'll check this out.
